File: DailyFlash Solutions/DailyFlash Solutions/06-DailyFlash_Solutions/20_Jan_Solutions_One/Python/pattern.py
"""
Program 5: Write a Program to print following Pattern.
Output:
0
1 0
1 0
1 0
1
0 0 0 0
1 1 1 1

"""


for outrItr in range(4):
    for innerItr in range(4):
        if outrItr%2==0:
            print("0",end=" ")
        else:
            print("1",end=" ")
    print(" ")    

File: DailyFlash Solutions/DailyFlash Solutions/06-DailyFlash_Solutions/20_Jan_Solutions_One/JAVA/Pattern.java
/*
Program 5: Write a Program to print following Pattern.
Output:
0
1 0
1 0
1 0
1
0 0 0 0
1 1 1 1

*/

class Pattern{
	
	public static void main(String args[]){
		
		for(int outritr = 0 ; outritr < 4 ; outritr++ ){

			for(int inneritr = 0 ; inneritr < 4 ; inneritr++ ){

				if(outritr%2==0){  
					System.out.printf("0 ");
				}else{
					System.out.printf("1 ");
				}
			}
			System.out.printf("\n");
		}
	}	
}	
